File: test/llvm_test_code/linear_constant/global_07.cpp
int g = 0;

int foo(int a) {
  a += g;
  return a;
}

int bar(int b) {
  g += 1;
  return b + 1;
}

int main() {
	g += 1;
  int i = 0;
  i = foo(10);
  i = bar(3);
	return 0;
}

File: test/llvm_swift_test_code/linear_constant/global_07.swift
var g = 0

@main
struct MyMain {

  static func foo (_ a: Int)-> Int {
    var x = a
    x += g
    return x
  }

  static func bar (_ b: Int)-> Int {
    g += 1
    return b + 1
  }

    static func main() {
      g += 1
      var i = 0
      i = foo(10)
      i = bar(3)
    }
}
